### DISCUSSION TITLE
Projects beta - ability to paste images / screenshots

### DISCUSSION THREAD

**Discussion by asgeo1** [community/community#13188] [Projects and Issues]
The new projects beta, I like it, but it's missing the ability to copy/paste images into the body of the text.

It should work exactly the same as how Issues work, you can take a screenshot and paste it in.

Currently as a work around, I have to convert every project beta task to an "Issue" first, just so I get this feature, which is kind of annoying.

**Comment by tristanlabbe** [comment]
Yes please!!

**Comment by rmcsharry** [comment]
![image](https://user-images.githubusercontent.com/1156100/162616115-c1e55653-2129-41a0-9842-a4e1babecc27.png)

**Comment by rmcsharry** [comment]
> The new projects beta, I like it, but it's missing the ability to copy/paste images into the body of the text.
> 
> It should work exactly the same as how Issues work, you can take a screenshot and paste it in.
> 
> Currently as a work around, I have to convert every project beta task to an "Issue" first, just so I get this feature, which is kind of annoying.

![image](https://user-images.githubusercontent.com/1156100/162616186-de597320-74c3-49b3-9ed5-0aacd500e5a3.png)

**Comment by Sammeeey** [comment]
### This feedback is a duplicate of #9917 but has a slightly clearer description.
@asgeo1 maybe you could edit your initial feedback to inlcude a link to #9917 ?
Then we would have a higher chance of succes, if @davidjb99 improves the description there?
### Visualization of issue
- manual insertion of images (with URLs) possible
- copy+paste insertion (without URLs) impossible
![grafik](https://user-images.githubusercontent.com/49591562/164958924-d23696a4-f241-4b56-9525-dd5e2a47f007.png)
![grafik](https://user-images.githubusercontent.com/49591562/164959033-58f1b71b-dd9a-4797-b7b4-9f0a8e63f2bc.png)


### Why is this an issue?
Sometimes you just have images/screenshots in the clipboard (which don't have URL's).
Hence we need a solution to insert these images too✌
